<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>upgrade_tool</class>
 <widget class="QMainWindow" name="upgrade_tool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>upgrade_tool</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="COM_SETTING">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>411</width>
      <height>111</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>COM Setting：</string>
    </property>
    <widget class="QPushButton" name="COM_SET_REFRESH">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh COM</string>
     </property>
    </widget>
    <widget class="QPushButton" name="COM_SET_OPEN">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>70</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>32</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string> Index：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>31</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>BaudRate：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="COM_SET_INDEX">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="COM_SET_BR">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>411</width>
      <height>431</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>1</number>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="BBFW_BURN">
     <attribute name="title">
      <string>BBFW UPGRADE</string>
     </attribute>
     <widget class="QProgressBar" name="BBFW_BURN_PROBAR">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>381</width>
        <height>23</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>UpArrowCursor</cursorShape>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="BBFW_BURN_START">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>360</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>BB:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="BBFW_BURN_DIR">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>321</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QToolButton" name="BBFW_BURN_TOOL">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>170</y>
        <width>21</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>73</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Auto HDR</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Flash Addr：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="FLASH_BURN_ADDR">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>0x300000</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="SYS_BURN">
     <attribute name="title">
      <string>SYSTEM UPGRADE</string>
     </attribute>
     <widget class="QPushButton" name="SYS_BURN_START">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>360</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="SYS_BURN_MBR_PROBAR">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>381</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="SYS_BURN_AC_PROBAR">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>381</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="SYS_BURN_BL_PROBAR">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>381</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="SYS_BURN_BB_PROBAR">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>381</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="SYS_BURN_MBR_DIR">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>MBR:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="SYS_BURN_AC_DIR">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>AC:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="SYS_BURN_BL_DIR">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>200</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>BL:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>BB:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="SYS_BURN_BB_DIR">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QToolButton" name="SYS_BURN_MBR_TOOL">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>21</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QToolButton" name="SYS_BURN_AC_TOOL">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>110</y>
        <width>21</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QToolButton" name="SYS_BURN_BL_TOOL">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>200</y>
        <width>21</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QToolButton" name="SYS_BURN_BB_TOOL">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>280</y>
        <width>21</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>412</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
